### ISSUE TITLE
Ubuntu package nautilus integration not working

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/desktop#583]
I recently changed from https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client to https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client-beta and since the beta2 update the nautilus integration is not working anymore.

Also the nextcloud binary has no beautiful icon anymore :crying_cat_face: 

cc @ivaradi 

anything wrong with?
```
sudo apt install nextcloud-client nextcloud-client-nautilus
```

**Comment by ivaradi** [comment]
The install command looks good to me.

I know the reason for the missing icon, and will hopefully provide a solution soon. (The ```.desktop``` file refers to the icon with a small ```n```, but the icon file's name starts with a capital ```N```.)

However, I installed the beta client in an Ubuntu 16.04 VM, and the Nautilus installation worked for me, although I had to restart Nautilus after installation. Have you also restarted Nautilus (or rebooted the machine)? Which Ubuntu distribution are you using?

**Comment by nickvergessen** [comment]
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```

I rebooted my machine after restarting nautilus did not help.

Any more info I can help with?

**Comment by goyodiaz** [comment]
Seems to be working here, same OS and PPA.

**Comment by ivaradi** [comment]
As you can see above, I have created a pull request, that should fix the icon issue. Please, monitor the request. Once it is merged, the Ubuntu package will be created automatically.

About the Nautillus integration: I just tested installing the client into a vanilla Bionic install in a VirtualBox VM, and the integration worked right away (without even restarting anything) for me too. So it must be some local issue.

Could you try killing the Nautilus process (nautilus-desktop) and then start it in a terminal? It should normally print a message like ```Initializing Nextcloud-client-nautilus extension```, but in your case it might print an error message, that could help resolving the issue.

**Comment by nickvergessen** [comment]
Well the error here might actually explain it:
```
sys:1: PyGIWarning: Nautilus was imported without specifying a version first. Use gi.require_version('Nautilus', '3.0') before import to ensure that the right version gets loaded.
Initializing Nextcloud-client-nautilus extension
Using python version sys.version_info(major=2, minor=7, micro=15, releaselevel='candidate', serial=1)
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gi/overrides/GLib.py", line 754, in <lambda>
    func_fdtransform = lambda _, cond, *data: callback(channel, cond, *data)
  File "/usr/share/nautilus-python/extensions/syncstate-Nextcloud.py", line 147, in _handle_notify
    for line in self.get_available_responses():
  File "/usr/share/nautilus-python/extensions/syncstate-Nextcloud.py", line 137, in get_available_responses
    return data.decode().split('\n')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 147: ordinal not in range(128)
RuntimeError: object at 0x7f498c579a50 of type FolderColorMenu is not initialized
RuntimeError: object at 0x7f498c579a50 of type FolderColorMenu is not initialized
RuntimeError: object at 0x7f498c579a50 of type FolderColorMenu is not initialized
Nautilus-Share-Message: 16:33:15.108: Called "net usershare info" but it failed: Kindprozess »net« konnte nicht ausgeführt werden (No such file or directory)
RuntimeError: object at 0x7f498c579a50 of type FolderColorMenu is not initialized
RuntimeError: object at 0x7f498c579a50 of type FolderColorMenu is not initialized
```

**Comment by ivaradi** [comment]
Indeed, the error code indicates that the path of your sync folder most likely contains some non-ASCII characters, which are not handled correctly by the plugin. This should be fixed in #652.

**Comment by nickvergessen** [comment]
Thanks works now in hte latest beta again :+1:
